tests/test.login.spec.ts

```diff
@@ -0,0 +1,176 @@
+    const homepage = new HomePage(page);
+
+    await homepage.fillInput('email', validEmail);
+    await homepage.checkInputValue('email', validEmail);
+    await homepage.fillInput('password', validPassword);
+    await homepage.checkInputValue('password', validPassword);
+
+    await homepage.clickOnHidePasswordIcon();
+    await homepage.checkPasswordInputType('shown');
+    await homepage.clickOnHidePasswordIcon();
+    await homepage.checkPasswordInputType('hidden');
+
+    await homepage.clickOnSubmitLoginFormBtn();
+    await homepage.checkUserIconIsDisplayed(true);
+    await homepage.checkUrl(homepageUrl);
+
+    await homepage.clickOnUserIcon();
+    await homepage.checkProfileDropDownIsDisplayed();
+    await homepage.checkProfileDropDownEmail(validEmail);
+    await homepage.logout();
+    await homepage.checkUserIconIsDisplayed(false);
+})
+
+test('test case C202: Authorization with valid phone and password', async( {page} ) => {
+    const homepage = new HomePage(page);
+    const profilePage = new ProfilePage(page);
+
+    const correctPhoneNumbers: string[] = [
+        process.env.CORRECT_PHONE_NUMBERS_FULL || '',
+        process.env.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS || '',
+        process.env.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS38 || ''
+    ];
+
+    for(const phoneNumber of correctPhoneNumbers) {    
+        await homepage.fillInput('email', phoneNumber);
+        await expect(await homepage.checkInputError('email')).toBe(false);
+        await homepage.fillInput('password', validPassword);
+        await expect(await homepage.checkInputError('password')).toBe(false);
+
+        await homepage.clickOnSubmitLoginFormBtn();
+        await homepage.checkUserIconIsDisplayed(true);
+        await homepage.checkUrl(homepageUrl);
+
+        await homepage.clickOnUserIcon();
+        await homepage.clickOnMyProfileMenuItem();
+        await profilePage.checkUrl('/owner-cabinet/');
+        await profilePage.checkPhoneInputIsDisplayed();
+        await profilePage.checkPhoneInputValue(validPhone);
+        await profilePage.clickOnLogoutBtn();
+        await homepage.checkUrl(homepageUrl);
+
+        await homepage.clickOnEnterBtn();
+    }
+})
+
+test('test case C207: Authorization with invalid phone', async( { page } ) => {
+    const homepage = new HomePage(page);
+
+    const incorrectPhoneNumbers: string[] = [
+        process.env.INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITHOUTLN || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITH_DASHES || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITH_SPACES || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITH_BRACKET || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITHOUT_COUNTRY_CODE_BRACKETS || '',
+        process.env.INCORRECT_PHONE_NUMBERS_ELEVEN_DIGITS || '',
+        process.env.INCORRECT_PHONE_NUMBERS_OTHER_COUNTRY_CODE || '',
+        process.env.INCORRECT_PHONE_NUMBERS_WITHOUT_38 || ''
```

Comment:
These should not be stored as env.vars. A simple data file should be enough here